DepositeCalcTests/Pages/LoginPage.cs

```diff
@@ -39,4 +41,75 @@ public bool IsOpened()
             return isLoginFldPresent;
         }
     }
+
+    public class RemindPasswordView
+    {
+        private IWebDriver driver;
+        private IWebElement openButton;
+        public RemindPasswordView(IWebDriver driver, IWebElement openButton)
+        {
+            this.driver = driver;
+            this.openButton = openButton;
+        }
+
+        private IWebElement RemindPasswordForm => driver.FindElement(By.XPath("//iframe[@id='remindPasswordView']"));
+        private IWebElement CloseBtn => driver.FindElement(By.XPath("//button[text()='x']"));
+        public bool IsShown => RemindPasswordForm.Displayed;
+        private IWebElement EmailFld => driver.FindElement(By.XPath("//input[@placeholder='Email address']"));
+        private IWebElement SendBtn => driver.FindElement(By.XPath("//button[text()='Send']"));
+
+        public string ErrMessageForRemindPasswordForm
+        {
+            get
+            {
+                var locator = By.Id("message");
+                IWebElement errMessageForRemindPasswordForm = driver.FindElement(locator);
+                new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => errMessageForRemindPasswordForm.Text.Length > 0);
+                return errMessageForRemindPasswordForm.Text;
+            }
+        }
+
+        public void Open()
+        {
+            openButton.Click();
+        }
+
+        public void Close()
+        {
+            driver.SwitchTo().Frame("remindPasswordView");
+            CloseBtn.Click();
+            driver.SwitchTo().DefaultContent();
+        }
+
+        public (bool IsSuccessful, string Message) RemindPassword(string email)
+        {
+            driver.SwitchTo().Frame("remindPasswordView");
+            EmailFld.SendKeys(email);
+            SendBtn.Click();
+            if (IsAlertPresent())
+            {
+                IAlert alert = driver.SwitchTo().Alert();
+                var result = (true, alert.Text);
+                alert.Accept();
+                return result;
+            }
+            else
+            {
+                var result = (false, ErrMessageForRemindPasswordForm);
+                return result;
```

Comment:
You can simplify it a little:
```suggestion
                return (false, ErrMessageForRemindPasswordForm);
```